tests/src/Functional/CommentsGenerationTest.php

```diff
@@ -57,6 +57,21 @@ public function testThereIsNoNewLineAboveTagsSection(\ReflectionClass $mainClass
     {
         $property = $mainClass->getProperty('classmap');
         $this->assertContains('/**
-     * @var array', $property->getDocComment());
+     * @var array', $this->normalizeNEL($property->getDocComment()));
+    }
+
+    /**
+     * Normalizes the newline.
+     *
+     * Windows uses CR+LF for the newline.
+     * This method converts the newline to UNIX style.
+     *
+     * @param string $string
+     *
+     * @return string
+     */
+    private function normalizeNEL($string)
+    {
+        return str_replace("\r", '', $string);
```

Comment:
This is issue of generation code. Classes of namespace `Wsdl2PhpGenerator\PhpSource` used `PHP_EOL` constant, but in tests always used `\n` for the newline.

I don't know as this decide other manner.